package.json

```diff
@@ -30,7 +30,8 @@
     "lint-staged": "^15.1.0",
     "prettier": "^3.1.0",
     "simple-git-hooks": "^2.9.0",
-    "typescript": "^5.3.3"
+    "typescript": "^5.3.3",
+    "viem": "^1.20.3"
```

Comment:
Do we need viem here (at the top level)?

Reply:
I believe it helps with resolving the specific version instead of letting pnpm pick the version. I was using wagmi + viem as a reference for all of this, and they do the same.